metricbeat/module/kubernetes/state_service/_meta/testdata/kube-state-metrics.1.7.plain-expected.json

```diff
@@ -0,0 +1,86 @@
+[
+    {
+        "event": {
+            "dataset": "kubernetes.service",
+            "duration": 115000,
+            "module": "kubernetes"
+        },
+        "kubernetes": {
+            "namespace": "default",
+            "service": {
+                "cluster_ip": "10.96.0.1",
+                "created": "2019-07-19T10:39:29.000Z",
+                "labels": {
+                    "label_component": "apiserver",
+                    "label_provider": "kubernetes"
+                },
+                "name": "kubernetes",
+                "type": "ClusterIP"
+            }
+        },
+        "metricset": {
+            "name": "state_service",
+            "period": 10000
+        },
+        "service": {
+            "address": "127.0.0.1:55555",
+            "type": "kubernetes"
+        }
+    },
+    {
+        "event": {
+            "dataset": "kubernetes.service",
+            "duration": 115000,
+            "module": "kubernetes"
+        },
+        "kubernetes": {
+            "namespace": "kube-system",
+            "service": {
+                "cluster_ip": "10.96.0.10",
+                "created": "2019-07-19T10:39:31.000Z",
+                "labels": {
+                    "label_k8s_app": "kube-dns",
```

Comment:
Wondering if it would be better to have these labels without the prefix `label_`. Would this be possible and of sense?

Reply:
if would. some kind of processor inside prometheus helper could allow for transformation of _non mapped labels_.
The reason for not doing that right now are:
- prometheus helper transform prometheus resources at the source into an structure that loses track of labels/metrics relationship. We should propose a different way of dealing with it
- as part of that refactor, duplicated metric families, non mapped labels and such, can be more natively targeted. 

If you think otherwise for this PR, I'm very glad to chat upon it.

Reply:
+1 in ignoring this for now.